### ISSUE TITLE
Follow returns success but failed

### ISSUE THREAD

**Issue by itemimg** [brunocvcunha/instagram4j#177]
Hi, I am running Instagram4j with about 4 accounts for about 5 months now and it's great!

Since yesterday I experience the issue, that the FollowRequests doesn't work anymore. Instagram4j doesn't throw any error, but if I check if the user has been followed for real, I realized that it didn't.

It happens for all 4 accounts at the same time with different FollowRequests.

Did Instagram change something or may I been blocked temporarily?

Best regards!

**Comment by ChrisJSt** [comment]
Hey man,

I was running it for multiple accounts a few months ago and I had experienced the same issue with one particular account. There would be no indicator of a failure or even a spam message, but when I tried to send an InstagramFollowRequest the subsequent message would say following:"false". After some time of waiting I would be able to follow again with that account for about 20 follows until the same issue persisted.
